lib/index.js

```diff
@@ -27,6 +28,7 @@ exports.agents = {
 exports.request = function (method, url, options, callback, _trace) {
 
     options = options || {};
+    var logger = new Logger();
```

Comment:
I tried it a different way (doing a new during the export) but was unable to turn the logging on and off for the tests using any other approach. This is the only thing that I could get to work.  If you have another idea, I'd be happy to try it; everything I could think of has been a dead end.